src/kernel/checks.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <kernel/checks.h>
+
+#include <compat/sanity.h>
+#include <key.h>
+#include <random.h>
+#include <util/time.h>
+
+namespace kernel {
+
+std::optional<SanityCheckError> SanityChecks(Context&)
```

Comment:
Why bother introducing an argument if it is going to be unused?

Reply:
> Why bother introducing an argument if it is going to be unused?

It's because src/kernel/ is supposed to provide a library API, and the API should not have to change as we remove globals internally.

Reply:
Going to mark this as resolved, feel free to reopen if not!